README.md

```diff
@@ -27,6 +27,16 @@ def deps do
 end
 ```
 
+Please note that `ex_dtls` uses OpenSSL under the hood.
+We use `pkg-config` to search for include and lib files so you must make sure
+that `openssl.pc` file is searchable by `pkg-config`.
+If it is not in the default `pkg-config` search path, locate your `openssl.pc` and add its parent directory to the `PKG_CONFIG_PATH`.
+For example:
+
+```sh
+ export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:/opt/homebrew/Cellar/openssl@3/3.1.0/lib/pkgconfig
```

Comment:
Using `$(brew --prefix openssl@3)/lib/pkgconfig` has the advantage that it keeps working when the openssl is updated

Reply:
Done :+1: